tests/Unit/Evolution/Systems/Cce/Actions/Test_AnalyticBoundaryCommunication.cpp

```diff
@@ -152,8 +148,7 @@ struct test_metavariables {
 SPECTRE_TEST_CASE(
     "Unit.Evolution.Systems.Cce.Actions.AnalyticBoundaryCommunication",
     "[Unit][Cce]") {
-  Parallel::register_derived_classes_with_charm<
-      Cce::Solutions::WorldtubeData>();
+  PUPable_reg2(Cce::Solutions::RotatingSchwarzschild, "RotatingSchwarzschild");
```

Comment:
Maybe `Parallel::register_classes_in_list` is a bit safer to use because you don't need the explicit string. @wthrowe and I were thinking to add a `Parallel::register_classes<A, B, ...>()` overload to make this even easier. Up to you.